src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -259,6 +259,11 @@ internal void InitializationCompleted()
 		{
 			SystemThemeHelper.SystemThemeChanged += OnSystemThemeChanged;
 			_initializationComplete = true;
+
+#if !HAS_UNO_WINUI
```

Comment:
Is the behavior not consistent between UWP and WinUI? or otherwise why is this conditional?

Reply:
Actually the method and even the event args don't exist in WinAppSDK anymore. I made a breaking changes PR for that, but I will rebase it on the changes in this PR after it is merged to breaking changes branch 👍